src/002_installation.md

```diff
@@ -55,48 +55,49 @@ Continue as described in Section [Running Tamarin](#sec:running-tamarin) to run
 Mac OS X {#sec:macosx}
 --------
 
-### Automatic Installation of the Tamarin binary {#sec:MacOSBinInstall}
+### Automatic Installation of Tamarin {#sec:MacOSBinInstall}
 
-The fastest way to install Tamarin on Mac OS X is to use [Homebrew](http://brew.sh/) [Cask](https://caskroom.github.io/).
+The fastest way to install Tamarin on Mac OS X is to use [Homebrew](http://brew.sh/).
 
 If you already have this installed, it is as simple as running the following two commands in your terminal:
 
   * `brew update`
-  * `brew cask install tamarin-prover`
+  * `brew install homebrew/science/tamarin-prover`
 
 You can now run Tamarin from the command line by typing `tamarin-prover`.
 Continue as described in Section [Running Tamarin](#sec:running-tamarin) to
-run Tamarin for the first time. (Warning: the previous stable version of Tamarin
-(v1.0.0) does not realize that Maude v2.7.1 is more than sufficient; it will
-currently give you a warning when you run `tamarin-prover test` but it is safe
-to ignore this.)
+run Tamarin for the first time.
 
-#### Installing Homebrew & Cask
-If you don't have Homebrew and/or Cask installed:
+#### Installing Homebrew
```

Comment:
I think having all these instructions is confusing. Should both sections now be collapsed into one? Have a TL;DR at top of the section then include this detail below?